ACME-PS/internal/classes/crypto/Certificate.ps1

```diff
@@ -29,18 +29,51 @@ class Certificate {
         }
     }
 
+    static [Security.Cryptography.X509Certificates.X509Certificate2Collection] ConvertToCertificateCollection([System.Collections.Generic.List[Security.Cryptography.X509Certificates.X509Certificate2]] $acmeCertificates) {
+        $result = [Security.Cryptography.X509Certificates.X509Certificate2Collection]::new();
+        $todo = [System.Collections.Generic.List[Security.Cryptography.X509Certificates.X509Certificate2]]::new();
```

Comment:
Same as below `[Collections.ArrayList]::new()` should be enough.
You might also consider using the parameter $acmeCertificates 'directly' as $todo and skip the first foreach

Reply:
ps. Would you also suggest changing the method signature (i.e. to `ConvertToCertificateCollection([Collections.ArrayList] $acmeCertificates)`), or would you prefer to enforce this as being some kind of X509 Certificate enumerable?

Reply:
I'd consider it a "private" method, so it does not need to check too much here.
if you want you can try making it a "hidden static" (or static hidden?)

Reply:
Good shout; thanks.  Done.